src/context/schemas/position.schema.json

```diff
@@ -1,12 +1,12 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "$id": "https://fdc3.finos.org/schemas/1.0/position.schema",
+  "$id": "https://fdc3.finos.org/schemas/next/position.schema",
   "type": "object",
   "allOf": [{ "$ref": "context.schema#/" }],
   "properties": {
-    "type": { "enum": ["fdc3.position"] },
+    "type": { "const": "fdc3.position" },
     "instrument": { "$ref": "instrument.schema#/" },
     "holding": { "type": "number", "minimum": 0 }
   },
-  "required": ["instrument", "holding"]
+  "required": ["instrument"]
```

Comment:
It doesn't make sense to me that the `holding` property on position is optional, which is why it was previously required, but according to https://fo.finos.org/docs/objects/position it is optional.

Reply:
agreed.  the object is not meaningful without a holding value.

Reply:
I will change it back.